src/extension.ts

```diff
@@ -31,13 +33,13 @@ export function activate(context: vscode.ExtensionContext) {
 
     context.subscriptions.push(disposable1);
 
-    const disposable2 = vscode.commands.registerCommand('ucl-rsdg.validateAtf', () => {
+    const disposable2 = vscode.commands.registerCommand('ucl-rsdg.validateAtf', async () => {
         const editor = vscode.window.activeTextEditor;
-        const filePath = editor.document.uri.fsPath;
+        const fileName = basename(editor.document.uri.fsPath);
         const fileProject = "cams/gkab";
         const fileContent = editor.document.getText();
         // The validate function is currently not mapped to the appropriate logging functions
-        const result = validate(filePath,fileProject,fileContent);
+        const result = await validate(fileName,fileProject,fileContent);
```

Comment:
Yes, it only needs the name. It appends that filename to a path on the server, so before it was ending up with a weird path. It doesn't seem to matter in the end because the new path gives an error anyway (same as in Nammu) but it seems that the intention is for the basename.